WcaOnRails/app/javascript/edit-events/ButtonActivatedModal.jsx

```diff
@@ -28,7 +28,15 @@ export default class ButtonActivatedModal extends React.Component {
         {this.props.buttonValue}
         <Modal show={this.state.showModal} onHide={this.close}>
           <form className={this.props.formClass}
-                onSubmit={e => { e.preventDefault(); this.props.onOk(); }}
+                onSubmit={e => {
+                  // Because we're rendering a modal inside of a modal, we're
+                  // actually also rendering a form inside of a form. We don't
+                  // want submitting this inner form to trigger a submit of the
+                  // outer form, so we must stop event propagation here.
```

Comment:
Quite a story :smiley:

Reply:
Ha! Yeah. We should probably not be doing a modal on top of a modal, but this is the easy fix for now.